x/ccv/provider/keeper/relay_test.go

```diff
@@ -676,3 +676,40 @@ func TestHandleVSCMaturedPacket(t *testing.T) {
 	_, found = pk.GetUnbondingOpIndex(ctx, "chain-1", 3)
 	require.False(t, found)
 }
+
+// TestSendVSCPacketsToChainFailure tests the SendVSCPacketsToChain method failing
+func TestSendVSCPacketsToChainFailure(t *testing.T) {
+	// Keeper setup
+	providerKeeper, ctx, ctrl, mocks := testkeeper.GetProviderKeeperAndCtx(t, testkeeper.NewInMemKeeperParams(t))
+	defer ctrl.Finish()
+	providerKeeper.SetParams(ctx, providertypes.DefaultParams())
+
+	// Append mocks for full consumer setup
+	mockCalls := testkeeper.GetMocksForSetConsumerChain(ctx, &mocks, "consumerChainID")
+
+	// Set 3 pending vsc packets
+	providerKeeper.AppendPendingVSCPackets(ctx, "consumerChainID", []ccv.ValidatorSetChangePacketData{{}, {}, {}}...)
+
+	// append mocks for the channel keeper to return an error
+	mockCalls = append(mockCalls,
+		mocks.MockChannelKeeper.EXPECT().GetChannel(ctx, ccv.ProviderPortID,
+			"CCVChannelID").Return(channeltypes.Channel{}, false).Times(1),
```

Comment:
.Times(1) seems unnecessary
```suggestion
			"CCVChannelID").Return(channeltypes.Channel{}, false),
```

Reply:
Hmm I don't see the reason to remove the `Times(1)` since we indeed only want this method called once. Better to be explicit with the mocking package imo